spec/craft_spec.rb

```diff
@@ -0,0 +1,24 @@
+require './lib/person'
+require './lib/craft'
+require './lib/event'
+
+RSpec.describe Craft do
+  let(:craft) {Craft.new('knitting', {yarn: 20, scissors: 1, knitting_needles: 2})}
+  it "exisits" do
+    actual = craft
+    expected = Craft
+    expect(actual).to be_a(expected)
+  end
+  describe '#attributes' do
+    it 'has a name' do
+      actual = craft.name
+      expected = 'knitting'
```

Comment:
I like this use of `actual` and `expected` as it make it really clear what you are testing